PocketKit/Sources/Sync/PocketSource.swift

```diff
@@ -121,6 +121,18 @@ public class PocketSource: Source {
             completion()
         }
     }
+
+    public func saveFromExtension(url: URL) {
```

Comment:
This function gets called from the extension. All it does is store something in core data. The main app can then query for these items and perform any actions necessary (see below)